package.json

```diff
@@ -76,6 +76,7 @@
     "test:build": "vitest --mode test-build",
     "tsd": "tsc -p test-dts/tsconfig.tsd.json",
     "build": "rollup -c rollup.config.js",
+    "prepare": "rollup -c rollup.config.js",
```

Comment:
We're duplicating `build` script instead of `pnpm run build` - we do not want to rely on `pnpm`, and having `build` invoked as `pnpm run prepare` feels very wrong to me